libs/widget-lib/src/types.ts

```diff
@@ -199,10 +199,17 @@ export interface CowSwapWidgetParams {
    * Network ID.
    */
   chainId?: SupportedChainId
+
+  /**
+   * An id of a network to bridge the output token.
+   */
+  targetChainId?: number
```

Comment:
We can add a TODO: to set it to `TargetChainId` which was added in this PR https://github.com/cowprotocol/cow-sdk/pull/251/files#diff-0ece96aace63d57c02cd7d5b601e8e26f8c57051998cf9cffb2aaa4739dd35d3R31

Reply:
I have a concern about that:
https://github.com/cowprotocol/cow-sdk/pull/251#discussion_r2001056674